content/en/blog/news/outreachy-blog2-2020.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Outreachy - Motivation to apply! "
+date: 2020-01-22
+draft: false
+description: >
+  Let's contribute to open-source - Get, set, GO!
+categories: [blog]
+keywords: [Community, 2020, Outreachy]
+slug: "outreachy2019"
+aliases: []
+author: Neha Gupta
+---
+<figure>
+<img src="/images/community/events/outreachy.png"/>
+</figure>
+
+### Outreachy open-source contribution for applicants — Asking/Finding for help
+
+This blog might be helpful for beginners who are fear-stricken or I would say hesitant to ASK, to get lost in the new world while trying to understand any open source project, fear of asking questions that may sound stupid later on or are very obvious!
+First of all..
+Relax!
+Everyone starts from somewhere and has a learning curve!..
+There are some pre-requisites that may help you get into open-source development better..
```

Comment:
These are great tips you list here! It would be great to highlight the information visually as bullet points or a list.

Reply:
Makes sense   :)